lib/ace/lib/event.js

```diff
@@ -115,6 +115,29 @@ exports.capture = function(el, eventHandler, releaseCaptureHandler) {
     return onMouseUp;
 };
 
+exports.addTouchMoveListener = function (el, callback) {
+    if ("ontouchmove" in el) {
+        var startx, starty;
+        exports.addListener(el, "touchstart", function (e) {
+            var touchObj = e.changedTouches[0];
+            startx = parseInt(touchObj.clientX);
+            starty = parseInt(touchObj.clientY);
+        });
+        exports.addListener(el, "touchmove", function (e) {
+            var factor = 1,
+            touchObj = e.changedTouches[0];
+
+            e.wheelX = -(parseInt(touchObj.clientX) - startx) / factor;
```

Comment:
>  then the browser is not following W3 standards 

then we don't need `parseInt` call here, right?

I can merge this once you fill the cla form as described in [CONTRIBUTING.md](https://github.com/ajaxorg/ace/blob/master/CONTRIBUTING.md)